src/main/java/com/endava/soa_5ed/task2/arrays/CodeReadingTask.java

```diff
@@ -0,0 +1,75 @@
+package com.endava.soa_5ed.task2.arrays;
+
+public class CodeReadingTask {
+    /* Arrays - Exercise 1 */
+    public static int multiplyArrayNumbers(int[] a) {
+        //this method multiplies all the numbers in an array and returns the product
+        int product = 1;
+        int i = 0;
+        while (i < a.length) {
+            product = product * a[i];
```

Comment:
you also may try product *= a[i]; to simplify your code.

Reply:
done